backend/typescript/models/book.model.ts

```diff
@@ -0,0 +1,55 @@
+/* eslint import/no-cycle: 0 */
+import {
+  Column,
+  DataType,
+  Model,
+  Table,
+  BelongsTo,
+  BelongsToMany,
+  ForeignKey,
+} from "sequelize-typescript";
+import { IntegerDataType, RangeDataType } from "sequelize/types";
+import Author from "./author.model";
+import BookAuthor from "./book_author.model";
+import BookPublisher from "./book_publisher.model";
+import Publisher from "./publisher.model";
+import Review from "./review.model";
+import Series from "./series.model";
+
+@Table({ tableName: "books" })
+export default class Book extends Model {
+  @ForeignKey(() => Review)
+  @Column({ type: DataType.INT })
+  review_id!: number;
+
+  @Column({ type: DataType.STRING })
+  title!: string;
+
+  @ForeignKey(() => Series)
+  @Column({ type: DataType.INT })
+  series_id?: number;
+
+  @Column({ type: DataType.STRING })
+  series_order?: string;
+
+  @Column({ type: DataType.ARRAY(DataType.STRING) })
+  illustrator?: string[];
+
+  @Column({ type: DataType.ARRAY(DataType.STRING) })
+  translator?: string[];
+
+  @Column({ type: DataType.ARRAY(DataType.JSON) })
+  formats!: DataType.JSON[];
+
+  @Column({ type: DataType.RANGE(DataType.INTEGER) })
+  age_range?: RangeDataType<IntegerDataType>;
+
```

Comment:
I think here we need to add a BelongsTo() to Review as well - sorry for not noticing this earlier, when you do this change please all add a HasMany() to Book in review.model.ts

Reply:
Done, thanks for pointing that out!